src/templates/entity/entity.template.ts

```diff
@@ -0,0 +1,124 @@
+import { allOfDereference } from '@templates/lib/allOfdereference'
+import { generateTsFile } from '@utils/generateTsFile'
+import { getFileImports } from '@utils/getFileImports'
+import * as _ from 'lodash'
+
+class EntityFileFactory {
+    private enums: Record<string, any> = {}
+
+    private imports: Record<string, Set<string>> = {
+        ['typeorm']: new Set(),
+        ['@nestjs/swagger']: new Set(),
+    }
+
+    private rootPath: string
+
+    private serviceName: string
+
+    private entities: { title: string; code: string }[]
+
+    constructor(rootPath: string, serviceName: string, entitySchemas: [string, any][]) {
+        this.rootPath = rootPath
+        this.serviceName = serviceName
+        this.imports['typeorm'].add('Column')
+        this.imports['typeorm'].add('Entity')
+        this.imports['typeorm'].add('PrimaryGeneratedColumn')
```

Comment:
```suggestion
        this.imports['typeorm'].add('Column')
        .add('Entity')
        .add('PrimaryGeneratedColumn')
```

Думаю можно так записывать и вщ вынести это из конструктора

Reply:
упростил, но оставил в конструкторе. нужно вроде как выполнить один раз...